opentelemetry-api/tests/__init__.py

```diff
@@ -11,12 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import pkg_resources
-
-# naming the tests module as a namespace package ensures that
-# relative imports will resolve properly for other test packages,
-# as it enables searching for a composite of multiple test modules.
-#
-# only the opentelemetry-api directory needs this code, as it is
-# the first tests module found by pylint during eachdist.py lint
-pkg_resources.declare_namespace(__name__)
```

Comment:
Was this the change that was discussed in the SIG that @aabmass had made previously? @ocelotl Removing this shouldn't have any affect right?

Reply:
No, what was discussed was [this line](https://github.com/open-telemetry/opentelemetry-python/blob/main/docs-requirements.txt#L8).

Removing this should not cause any problems, this file is redundant with this [line](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/pyproject.toml#L63).